lib/rustler_precompiled.ex

```diff
@@ -62,6 +62,11 @@ defmodule RustlerPrecompiled do
       Check the compatibiliy table between Elixir and OTP in:
       https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp
 
+    * `:retry` - A boolean that controls if downloads should be retried upon errors.
+      Defaults to `true`.
+
+    * `:retry_attempts` - The number of retry attempts before giving up. Defaults to `3`.
```

Comment:
Do you need two options? Maybe just `retry_attempts` and you can set it to 0 if you don't want a retry?

Reply:
Makes sense!

Reply:
max_retries is a decent name for this too, imo.